eng/WpfArcadeSdk/tools/GenApi.props

```diff
@@ -37,7 +40,8 @@
     <_GenerateReferenceAssemblySource>false</_GenerateReferenceAssemblySource>
     <_GenerateReferenceAssemblySource Condition="'$(GenerateReferenceAssemblySource)'=='true'
                                                         and '$(GenAPIEnabledProjects)'!=''
-                                                        and $(GenAPIEnabledProjects.Contains('$(MSBuildProjectName);'))">true</_GenerateReferenceAssemblySource>
+                                                        and $(GenAPIEnabledProjects.Contains('$(MSBuildProjectName);'))
+                                                        and !($(MSBuildProjectName.Contains('_wpftmp')))">true</_GenerateReferenceAssemblySource>
```

Comment:
How would this ever be true if you're checking `GenAPIEnabledProjects` contains the entire `$(MSBuildProjectName);`?